completions/bts

```diff
@@ -14,8 +13,7 @@ _comp_cmd_bts__cached_bugs()
 }
 
 # Generate APT source packages prefixed with "src:"
-# TODO:API: generator
-_comp_cmd_bts__src_packages_with_prefix()
+_comp_cmd_bts__compgen_src_packages_with_prefix()
 {
     local ppn=${cur:4} # partial package name, after stripping "src:"
     _comp_compgen -ac "$ppn" split -P "src:" -- \
```

Comment:
`-a` should be specified at the caller sides.

Reply:
True, but as this is an internal function and all call sites are expecting append semantics, I thought it'd be a bit simpler this way. Not that strong an opinion though, can change if you feel it would be better that way, or if I'm missing something.

Reply:
Actually, I'm not sure which of the append/replace semantics is better for the default behavior of generators, but I feel it is better to make the default behavior consistent among all the generators so we do not make mistakes by a wrong assumption on the semantics of a specific generator. Most of the generators are by default the replace semantics, so I thought it is consistent to make this also the replace semantics by default. It is clearer to have `-a` at the caller sides that the sequence of `_comp_compgen_*` there is intended for the append behavior.

Reply:
Move to #1028